### DISCUSSION TITLE
Click on blocks to add them in UI

### DISCUSSION THREAD

**Discussion by 0shuvo0** [GrapesJS/grapesjs#5511] [Q&A]
How can I make it so if I click on any blocks(specially on my Iphone) the block gets added to a in specific part the UI(maybe inside a special div like in between navbar and footer)?
Basically I am facing drag and drop not working/issues on phone.

**Comment by mosh-tudor** [comment]
@0shuvo0 
Have you tried https://github.com/GrapesJS/touch ?

**Comment by glaprida** [comment]
There is an option on `blockManager: {  appendOnClick: true } `
